mypyc/emitmodule.py

```diff
@@ -67,6 +67,14 @@ def encode_as_c_string(s: str) -> Tuple[str, int]:
     return '"{}"'.format(escaped), len(b)
 
 
+def encode_bytes_as_c_string(b: bytes) -> Tuple[str, int]:
+    """Produce a single-escaped, quoted C string and its size from a bytes"""
+    # This is a kind of abusive way to do this...
+    b = b.decode('unicode-escape').encode('raw-unicode-escape')
```

Comment:
Maybe do this decode/encode thing in genops, and use that as the key? (Since the result of that seems like it will be the correct byte string.)